src/transformers/pipelines/token_classification.py

```diff
@@ -234,56 +234,68 @@ def __call__(self, inputs: Union[str, List[str]], **kwargs):
             return answers[0]
         return answers
 
-    def set_subwords_label(self, entities: List[dict], strategy: str) -> dict:
-        def sub_words_label(sub_words: List[dict]) -> dict:
+    def set_subwords_label(self, entities: List[dict]) -> List[dict]:
+        strategy = self.subword_label_re_alignment
+
+        def set_labels(sub_words: List[dict]) -> dict:
             score = np.stack([sub["score"] for sub in sub_words])
-            if strategy == "default":
+            if strategy == "default" or strategy is True:
                 label_idx = score[0].argmax()
                 label = self.model.config.id2label[label_idx]
                 sub_words[0]["entity"] = label
+                sub_words[0]["score"] = score[0][label_idx]
                 for sub in sub_words[1:]:
                     sub["entity"] = "O"
-                return sub_words
+                    sub["score"] = 0.0  # what score should we assign here?
```

Comment:
Here the score is set to 0, because we are assigning the arbitrary label "O". Alternatively, we could use the corresponding score.

Reply:
I don't know which one is better, set to 0 or leave it as is??! 
but I think if we don't set the score to 0.0 then we should take for example `sub["score"].max()` or something like this here, because `sub["score"]` is a numpy array with scores for each type of entity.

Reply:
Yeah, I think taking the max would be confusing, because it wouldn't match the label. How about we set it to -1? So that it's clear it's not a meaningful score?